audio-worklet/src/subtractive-synth-demo.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+class SubtractiveSynthDemo {
```

Comment:
Sublime has an option to enforce space instead of tab. Please take a look and spend some time to set your editor: https://www.chromium.org/developers/sublime-text